projects/personachat/persona_seq2seq.py

```diff
@@ -32,24 +32,35 @@
 import torch.nn as nn
 import torch.nn.functional as F
 import torch
-import torchtext.vocab as vocab
-Glove = vocab.GloVe(name='840B', dim=300)
+try:
```

Comment:
nice

Reply:
thanks! yeah now you can run it from any directory too